client/app/requests/order-explorer/order-explorer.component.js

```diff
@@ -243,6 +243,7 @@ function ComponentController($filter, lodash, ListView, Language, OrdersState, E
     if (!refresh) {
       vm.loading = true;
     }
+    getFilterCount();
```

Comment:
I could probably chain these to where the filter count goes first and handles the other query in its success handler.   I thought about doing it but didnt want to shift too much code around but you have a point about multiple error messages.